packages/compose/src/hooks/use-focusable-iframe/index.js

```diff
@@ -3,25 +3,28 @@
  */
 import { useEffect } from '@wordpress/element';
 
+/**
+ * Dispatches a bubbling focus event when the iframe receives focus. Use
+ * `onFocus` as usual on the iframe or a parent element.
+ *
+ * @param {Object} ref React ref to the iframe.
+ */
 export default function useFocusableIframe( ref ) {
 	useEffect( () => {
-		const iframe = ref.current || ref();
-		const { ownerDocument } = iframe;
+		const { ownerDocument } = ref.current;
 		const { defaultView } = ownerDocument;
 		const { FocusEvent } = defaultView;
 
 		/**
 		 * Checks whether the iframe is the activeElement, inferring that it has
-		 * then received focus, and calls the `onFocus` prop callback.
+		 * then received focus, and dispatches a focus event.
 		 */
 		function checkFocus() {
-			if ( ownerDocument.activeElement !== iframe ) {
-				return;
+			if ( ownerDocument.activeElement !== ref.current ) {
```

Comment:
Code here doesn't match the comments above. It should be `===` here?

```suggestion
			if ( ownerDocument.activeElement === ref.current ) {
```